target/linux/ramips/dts/mt7621_unielec_u7621-01-16m.dts

```diff
@@ -15,6 +15,9 @@
 		reg = <0>;
 		spi-max-frequency = <14000000>;
 
+		#address-cells = <1>;
+		#size-cells = <1>;
```

Comment:
This fix doesn't make sense because `partitions` node doesn't have `reg` property. There is a PR try to silence these harmless warnings. https://github.com/openwrt/openwrt/pull/11302

Reply:
Fair enough, just annoying having these errors.

Will close this commit. Hopefully this gets the backported fixed  as mentioned